### PR TITLE
Doc: Updated Azure AD OAuth2 authentication topic.

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Updated topic based on this slack conversation: https://raintank-corp.slack.com/archives/C5PG2JK8W/p1632323887111100

### CODE REVIEW

**Review by zuchka — COMMENTED** [review]
lgtm re: the writing, but one of the Azure folks should sign off on the new tech steps 👍

**Review by sunker — APPROVED** [review]
Nice! Followed the instructions and it works as expected.

File: docs/sources/auth/azuread.md [hunk 1]
```diff
@@ -7,43 +7,44 @@ weight = 700
...
-1. Click **Endpoints** from the top menu.
+1. Note the **Application ID**. This is the OAuth client ID.
 
-   - Note the **OAuth 2.0 authorization endpoint (v2)**. This is the auth URL.
-   - Note the **OAuth 2.0 token endpoint (v2)**. This is the token URL.
+1. Click **Endpoints** from the top menu. 
 
-1. Click **Certificates & secrets** and then add a new entry under **Client secrets**.
+	- Note down the **OAuth 2.0 authorization endpoint (v2)** URL. This is the auth URL.
+	- Note the **OAuth 2.0 token endpoint (v2)**. This is the token URL.
```
Comment (GrafanaWriter) [inline docs/sources/auth/azuread.md:34]:
```suggestion
	- Note the **OAuth 2.0 authorization endpoint (v2)** URL. This is the authorization URL.
	- Note the **OAuth 2.0 token endpoint (v2)**. This is the token URL.
```

File: docs/sources/auth/azuread.md [hunk 2]
```diff
@@ -7,43 +7,44 @@ weight = 700
...
 
+1. Click **Certificates & secrets** and then add a new entry under **Client secrets** with the following configuration.
```
Comment (GrafanaWriter) [inline docs/sources/auth/azuread.md:36]:
```suggestion
1. Click **Certificates & secrets**, then add a new entry under **Client secrets** with the following configuration.
```

File: docs/sources/auth/azuread.md [hunk 3]
```diff
@@ -7,43 +7,44 @@ weight = 700
...
    - Description: Grafana OAuth
    - Expires: Never
 
-1. Click **Add**, and then copy the key value. This is the OAuth client secret.
+1. Click **Add** then copy the key value. This is the OAuth client secret.
 
-1. Click **Manifest**.
+1. Click **Manifest**, then define the required Application Role values for Grafana: Viewer, Editor, Admin. Otherwise, all users will have the Viewer role.
```
Comment (GrafanaWriter) [inline docs/sources/auth/azuread.md:42]:
```suggestion
1. Click **Manifest**, then define the required Application Role values for Grafana: Viewer, Editor, or Admin. If not defined, all users will have the Viewer role. 
```

**Review by GrafanaWriter — APPROVED** [review]
Nice update. A few nits, otherwise LGTM.

File: docs/sources/auth/azuread.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/auth/azuread.md:34]: OMG! TY :)

**Pull request merged**